### ISSUE TITLE
In local Traction setup, can't connect to the second Endorser (the bcovrin-dev one)

### ISSUE THREAD

**Issue by loneil** [bcgov/traction#1031]
To reproduce

- Start up local Traction with `docker compose up`
- Create a tenant through Tenant UI
- Go to Profile and try to connect to bcovrin-dev ledger

Connection stays in request state

In **traction-agent** logs can see a connection error

```
2024-02-28 10:48:48 2024-02-28 18:48:48,702 aries_cloudagent.transport.outbound.manager ERROR >>> Error when posting to: http://host.docker.internal:8001; Error: (<class 'aiohttp.client_exceptions.ClientConnectorError'>, ClientConnectorError(ConnectionKey(host='host.docker.internal', port=8001, is_ssl=False, ssl=True, proxy=None, proxy_auth=None, proxy_headers_hash=None), ConnectionRefusedError(111, "Connect call failed ('192.168.65.254', 8001)")), <traceback object at 0x7f40665cd800>); Re-queue failed message ...
```

Connecting to BCOVRIN-TEST endorser is working fine.

**Context about local endorsers:**
Docker compose env starts up 2 endorsers for testing out ledger switching and things. (I don't like the naming of these and the env vals...)

"Endorser":
ACAPY_GENESIS_URL=http://test.bcovrin.vonx.io/genesis

"Endorser 1":
ACAPY_GENESIS_URL_1=http://dev.bcovrin.vonx.io/genesis

**Comment by loneil** [comment]
Oh! good to know, thanks.

So if we want to maintain 2 local endorsers on ledgers we'd need to replace with another one

**Comment by swcurran** [comment]
Yes — I would think Candy Dev might be OK.

**Comment by loneil** [comment]
Reference: https://github.com/bcgov/DITP/issues/82

**Comment by esune** [comment]
Can we close this now that BCovrin Dev has been decommissioned?

**Comment by loneil** [comment]
I'll need to tweak some things to alias them differently I think here.

**Comment by esune** [comment]
Just peeking at the error again, not sure what port `8001` refers to since it doesn't seem to be anywhere in the codebase? Wondering if the error is a webhook delivery and the issue with BCovrin Dev is that it was NOT set to auto-endorse transactions?

**Comment by loneil** [comment]
@esune this is working now with changes from https://github.com/bcgov/traction/pull/1050
If that setup is all good we can close this

**Comment by esune** [comment]
Changes look good! Approved the PR, closed as resolved by #1050
